packages/synapse-react-client/src/components/ColorGradient/ColorGradient.tsx

```diff
@@ -41,4 +48,29 @@ export function getColorPalette(
   return { colorPalette, textColors }
 }
 
-export default getColorPalette
+export function getContrastColorPalette(
```

Comment:
Added a new function, since we may not want the highly-contrasting colors in all cases (e.g. TimelinePlot)